src/gui/base/ButtonN.js

```diff
@@ -104,6 +104,7 @@ export type ButtonAttrs = {
 	isSelected?: lazy<boolean>,
 	noBubble?: boolean,
 	staticRightText?: string,
+	preventDefault?: boolean // Button can trigger form submit in some cases, use this if that's not desired
```

Comment:
You already get event in `click`, just do `preventDefault` there

Reply:
can do, just thought it helped to make the ButtonAttrs instances look more streamlined